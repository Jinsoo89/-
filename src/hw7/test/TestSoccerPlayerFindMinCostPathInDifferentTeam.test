# Test to find minimum cost path on soccer players graph

# Load soccerPlayers.tsv
LoadGraph players soccerPlayers.tsv

# Test findMinCostPath
FindPath players C.Ronaldo L.Messi