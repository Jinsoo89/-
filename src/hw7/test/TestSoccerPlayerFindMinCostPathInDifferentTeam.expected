# Test to find minimum cost path on soccer players graph

# Load soccerPlayers.tsv
loaded graph players

# Test findMinCostPath
path from C.Ronaldo to L.Messi:
C.Ronaldo to D.Maria with weight 0.333
D.Maria to Neymar with weight 1.000
Neymar to L.Messi with weight 1.000
total cost: 2.333